test/custom_resource_crud/custom_resource_crud_suite_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2018-2021 Splunk Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package customresourcecrud
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+
+	"github.com/splunk/splunk-operator/test/testenv"
+)
+
+const (
+	// PollInterval specifies the polling interval
+	PollInterval = 5 * time.Second
+
+	// ConsistentPollInterval is the interval to use to consistently check a state is stable
+	ConsistentPollInterval = 200 * time.Millisecond
+	ConsistentDuration     = 2000 * time.Millisecond
+)
+
+var (
+	testenvInstance *testenv.TestEnv
+	testSuiteName   = "customresourcecrud-" + testenv.RandomDNSName(2)
```

Comment:
Shorten the name from **customresourcecrud** to **crcrud** would resolve the test failure. This is causing the failure ```
Unexpected error:
[1]       <*errors.errorString | 0xc000352600>: {
[1]           s: "both customresourcecrud-ai and 288c31f combined have exceeded 24 chars",
[1]       }
[1]       both customresourcecrud-ai and 288c31f combined have exceeded 24 chars
[1]   occurred```